turftopic/models/cluster.py

```diff
@@ -169,6 +174,7 @@ def __init__(
         self.feature_importance = feature_importance
         self.n_reduce_to = n_reduce_to
         self.reduction_method = reduction_method
+        self.components_ = None
```

Comment:
Hmmm I'm not sure I would do this.
Reason: scikit-learn detects whether something is fitted by looking at whether it has attributes that end with an underscore. (Kinda stupid and opaque, and I have made this error before as well).
I would probably do a `hasattr()` instead. (though no biggie)

Reply:
good point -- i've changed this both here and in GMM